.travis.yml

```diff
@@ -12,7 +12,6 @@ env:
     - FLAGS=""
 
 before_install:
-  - echo yes | pecl upgrade apcu
   - |
```

Comment:
I think we can also remove line 16 to 19

Reply:
I've removed the requested lines.